src/main/java/seedu/exercise/logic/commands/DeleteExerciseCommand.java

```diff
@@ -42,10 +45,23 @@ public CommandResult execute(Model model) throws CommandException {
         Exercise exerciseToDelete = lastShownList.get(targetIndex.getZeroBased());
         eventPayload.put(KEY_EXERCISE_TO_DELETE, exerciseToDelete);
         model.deleteExercise(exerciseToDelete);
+        updateStatistic(model);
         EventHistory.getInstance().addCommandToUndoStack(this);
         return new CommandResult(String.format(MESSAGE_DELETE_EXERCISE_SUCCESS, exerciseToDelete));
     }
 
+    /**
+     * Update statistic about deleted exercise.
+     */
+    private void updateStatistic(Model model) {
```

Comment:
you can consider placing this method in the model class and just call `model.updateStatistics()`.

Reply:
Done.